python/ray/monitor.py

```diff
@@ -219,6 +216,16 @@ def _run(self):
             # round of messages.
             time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
 
+    def update_event_summary(self):
+        # Periodically emit event summaries.
```

Comment:
Can you elaborate a bit on what are you updating? 
I am not sure the reader has any idea here what even summaries are. (from reading all the code I do but I would prefer future readers not to have to)